mmseg/models/losses/accuracy.py

```diff
@@ -2,12 +2,13 @@
 import torch.nn as nn
 
 
-def accuracy(pred, target, topk=1, thresh=None):
+def accuracy(pred, target, topk=1, thresh=None, ignore_index=255):
```

Comment:
I think it would be better to set `None` as default value of `ignore_index`

Reply:
Sure, I will do it with adding UT by tonight(KST). Thank you for the comment :)